noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/common.nr

```diff
@@ -30,48 +27,6 @@ fn validate_arrays(app_public_inputs: PrivateCircuitPublicInputs) {
     validate_array(app_public_inputs.unencrypted_logs_hashes);
 }
 
-// Validate all read requests against the historical note hash tree root.
```

Comment:
Moved to `private_validation_request_processor`, which will be run by the tail circuits and will verify reads for both settled and pending note hashes.